### PR TITLE
Move wear capabilities to values folder

### PR DESCRIPTION
**Pull request by oas004**
#### WHAT
Follow up on https://github.com/google/horologist/issues/2281#issuecomment-2349319397 and move wear file back to values folder

#### WHY
Due to https://github.com/google/horologist/issues/2281#issuecomment-2349319397

#### Checklist :clipboard:
- [ ] Add explicit visibility modifier and explicit return types for public declarations
- [ ] Run spotless check
- [ ] Run tests
- [ ] Update metalava's signature text files

### CODE REVIEW

**Comment by oas004** [comment]
@yschimke I didn't run a test on release target, but maybe we can do that if u want? :)

File: docs/datalayer-helpers-guide.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ phone.
 
 2. Add the capabilities
 
-   Add a `wear.xml` file in the `res/raw` folder with the following content. It needs to be
+   Add a `wear.xml` file in the `res/values` folder with the following content. It needs to be
```
Comment (yschimke) [inline docs/datalayer-helpers-guide.md:31]: The docs don't show the two files

res/values - the string-array
res/raw - the tools:keep

Can you split in the docs also?

**Comment by yschimke** [comment]
> @yschimke I didn't run a test on release target, but maybe we can do that if u want? :)

If you want to add that go for it. It might need a new target to be added to CI workflows.

File: docs/datalayer-helpers-guide.md (hunk 1 above)
Comment (oas004) [reply docs/datalayer-helpers-guide.md:31]: Yeah I'll try to split them up :) Good idea :)

**Comment by oas004** [comment]
> I do wonder whether the keep file if put in a library would also work for other apps using the library?

Maybe it's worth testing?

File: docs/datalayer-helpers-guide.md (hunk 1 above)
Comment (oas004) [reply docs/datalayer-helpers-guide.md:31]: How about this? Now I also added the NB notice to make people see that it is a different folder :)

Comment (yschimke) [reply docs/datalayer-helpers-guide.md:31]: I think the

```
`res/raw` folder due to https://issuetracker.google.com/issues/348688201
```

Needs to move down to the next paragraph.

Comment (oas004) [reply docs/datalayer-helpers-guide.md]: Oh yes you are right! 🤦 😅  Thanks! Fixing

**Pull request merged**